CourseSearchAPIHttpTrigger/helper.py

```diff
@@ -74,3 +85,34 @@ def group_courses_by_institution(search_results, counts, limit, offset):
     }
 
     return results
+
+
+def remove_conjunctions_from_searchable_fields(course, institution):
+    if course != "":
+        course = remove_conjunctions(course)
+
+    if institution != "":
+        institution = remove_conjunctions(institution)
+
+    return course, institution
+
+
+conjunctions = {
+    "&",
+    "and",
+    "for",
+    "in",
+    "the",
+    "with"
+}
+
+
+def remove_conjunctions(searchable_field):
+    dict_of_string_parts = searchable_field.split(" ")
```

Comment:
But this isn't a dict 🙂

Reply:
It was and then changed what the variable was doing without changing the name oops